.github/workflows/build-rust-cross-platform.yml

```diff
@@ -67,6 +70,7 @@ jobs:
         if: ${{ !contains(matrix.settings.target, 'musl') }}
         env:
           RUSTFLAGS: "-D warnings"
+          MACOSX_DEPLOYMENT_TARGET: "10.6" # allows using new macos runner versions while still supporting older systems
```

Comment:
Do we want to bump this to something more recent say 10.14? 10.6 is ancient.

Reply:
Updated in [23db403](https://github.com/bitwarden/sdk/pull/1034/commits/23db403c3d1d20c46fe730a7557408c41d54cb0e).